### ISSUE TITLE
Characters on lower level not visible

### ISSUE THREAD

**Issue by f1nutz** [6502-workshop/NA#7]
Is it possible to have lower map level NPC's visible from a higher map level?
ie Horse trader and horses would be visible from castle walls above them etc.

![image](https://user-images.githubusercontent.com/44733387/47956476-6861a000-df7b-11e8-9996-92a07dbc6610.png)

**Comment by 6502-workshop** [comment]
Thanks. I can think of several complications in doing this, but if there are only a couple instances maybe I can shoehorn it in. Will investigate further later stage.

**Comment by f1nutz** [comment]
Yes I figured it would be a tough one since it would affect the scheduling routine etc.
Maybe if it included only non human entities like the horses, birds etc  in non accessible areas it might give enough life to the area to make it not so desolate. 
As well maybe you could have the odd guard here and there who are not on a schedule? rather than trying to match exactly what is on the ground level.

**Comment by 6502-workshop** [comment]
If the NPCs aren't on schedules then that's easy to do. But, that seems equally as weird to me as not having them present at all.
